programs/budget/src/budget_processor.rs

```diff
@@ -36,7 +36,9 @@ fn apply_signature(
             if &payment.to == key {
                 budget_state.pending_budget = None;
                 contract_keyed_account.try_account_ref_mut()?.lamports -= payment.lamports;
```

Comment:
no `checked_sub_lamports`?

Reply:
My plan was to make the actual math replacements in a separate pr to keep things simpler. This pr is primarily the mechanism, adding lamports.rs now, etc. The next pr(s) will all be making groups of similar +/- math replacements that can be verified as a common group by whoever owns that section of code.

Reply:
Ok, saw += replaced, wasn't sure why -= wasn't as well